src/components/AddEOSAccountForm/AddEOSAccountForm.js

```diff
@@ -19,6 +19,7 @@ const AddEOSAccountForm = ({ callAPI, handleSubmit, submitting }) => (
               required
               component={renderField}
               type="text"
+              autoComplete="off"
             />
```

Comment:
from what I'm reading, we need this `autoComplete="off"` on the `<form/>` as well, as I believe there are some browsers that require both otherwise they ignore the setting.  May want to check on that (I could have old information)

Reply:
fine, @datchley.
done.

Reply:
Thanks, sweetie. <3